integration-tests/change-base-dir/src/test/java/it/WiremockPetStore.java

```diff
@@ -0,0 +1,44 @@
+package it;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+import java.util.Collections;
+import java.util.Map;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class WiremockPetStore implements QuarkusTestResourceLifecycleManager {
+
+    private WireMockServer wireMockServer;
+
+    @Override
+    public Map<String, String> start() {
+        wireMockServer = new WireMockServer(8887);
```

Comment:
Using a fixed port may result in flaky tests.

```suggestion
        wireMockServer = new WireMockServer(WireMockConfiguration.wireMockConfig().dynamicPort());
```

Reply:
I think he copied from another project. We should review the others too.

Reply:
I created an issue for that: https://github.com/quarkiverse/quarkus-openapi-generator/issues/180

Reply:
removed after @ricardozanini suggestion of using simple-openapi instead